### ISSUE TITLE
Cannot install on Mac OS Catalina

### ISSUE THREAD

**Issue by hzhang1902** [benchpressjs/benchpressjs#90]
I'm trying to install benchpressjs on Mac OS Catalina 10.15.6 and I'm getting this error:

```
> benchpressjs@2.0.7 install ~/Documents/NodeBB/node_modules/benchpressjs
> cd rust/benchpress-rs && npm install

npm WARN deprecated neon-cli@0.4.2: Please upgrade to 0.5.0

> benchpress-rs@0.1.0 install ~/Documents/NodeBB/node_modules/benchpressjs/rust/benchpress-rs
> node scripts/install

[benchpress] No compatible pre-built native module found! { platform: 'darwin', module_version: '72' }
[benchpress] Building native module from source...
neon ERR! spawn cargo ENOENT

Error: spawn cargo ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! benchpress-rs@0.1.0 build: `neon build --release`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the benchpress-rs@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2020-10-14T03_49_56_603Z-debug.log
[benchpress] FATAL: Fallback build failed. For more info, see https://github.com/benchpressjs/benchpressjs#manually-building-native-module
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! benchpress-rs@0.1.0 install: `node scripts/install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the benchpress-rs@0.1.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2020-10-14T03_49_56_657Z-debug.log
npm WARN nodebb-plugin-emoji-android@2.0.0 requires a peer of nodebb-plugin-emoji@^2.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN textcomplete.contenteditable@0.1.1 requires a peer of textcomplete@^0.14.2 but none is installed. You must install peer dependencies yourself.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! benchpressjs@2.0.7 install: `cd rust/benchpress-rs && npm install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the benchpressjs@2.0.7 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2020-10-14T03_49_57_267Z-debug.log
```

**Comment by hzhang1902** [comment]
I have neon-cli installed and I am able to check the version (0.5.0)

**Comment by hzhang1902** [comment]
Turns out you have to have rust already installed in order to have cargo. This is unexpected since I come from NodeBB and thought everything is in JS. This should be put into docs.

**Comment by pitaj** [comment]
It is in the README. If you click on "Install the required dependencies for Neon." it will instruct you to install Rust among other things.

**Comment by pitaj** [comment]
Hi! It looks like a misunderstood your issue. I didn't realize you meant that this caused the `npm install` to completely fail. I've published v2.0.8 which should fix this problem:

    npm install benchpressjs@2.0.8
